docker/startup.sh

```diff
@@ -2,6 +2,9 @@
 neo4j start
 sleep 100
 neo4j status
+
+export MIRA_DOMAIN=$1
```

Comment:
I think a more elegant way to do this would be in the Dockerfile. That way, the build itself would take care of creating the environmental variable.